test/unit/lib/cms/sts/assume_role_web_identity_test.rb

```diff
@@ -1,57 +0,0 @@
-# frozen_string_literal: true
-
-require 'test_helper'
-
-class CMS::STS::AssumeRoleWebIdentityTest < ActiveSupport::TestCase
-  setup do
-    File.stubs(:exist?).returns(true)
-    assume_role_response.class.any_instance.stubs(:expiration).returns(1.minute.from_now)
-    assume_role_response.stubs(:credentials).returns(sts_credentials)
-  end
-
-  test 'CMS::STS::AssumeRoleWebIdentity is a singleton' do
-    assert CMS::STS::AssumeRoleWebIdentity.ancestors.include?(Singleton), true
-  end
-
-  test '#identity_credentials raises an error if the web_identity_token_file does not exist' do
-    CMS::S3.stubs(:config).returns(web_identity_token_file: sts_auth_params[:web_identity_token_file])
-    File.stubs(:exist?).with(sts_auth_params[:web_identity_token_file]).returns(false)
-    CMS::STS::AssumeRoleWebIdentity.instance.instance_variable_set('@identity_credentials', nil)
-
-    assert_raises(CMS::STS::AssumeRoleWebIdentity::TokenNotFoundError) do
-      CMS::STS::AssumeRoleWebIdentity.instance.identity_credentials
-    end
-  end
-
-  test '#identity_credentials calls AWS to get an instance of STS credentials' do
-    CMS::S3.stubs(:config).returns(
-      region: sts_auth_params[:region],
-      role_arn: sts_auth_params[:role_arn],
-      role_session_name: sts_auth_params[:role_session_name],
-      web_identity_token_file: sts_auth_params[:web_identity_token_file]
-    )
-
-    Aws::AssumeRoleWebIdentityCredentials.expects(:new).with(sts_auth_params).returns(assume_role_response)
-
-    assert CMS::STS::AssumeRoleWebIdentity.instance.identity_credentials, sts_credentials
```

Comment:
this didn't originally work afaict, used stubbing client in `calls AWS to get an instance of STS credentials` test instead